jax/_src/nn/initializers.py

```diff
@@ -160,7 +170,8 @@ def normal(stddev: RealNumeric = 1e-2,
   def init(key: Array,
            shape: core.Shape,
            dtype: DTypeLikeInexact = dtype,
-           out_sharding=None) -> Array:
+           out_sharding: ShardingType = None,
+           ) -> Array:
```

Comment:
why on a new line? If it doesn't exceed the 80 character limit, can you put it on the previous line? here and everywhere else.

Reply:
Looks like you resolved it but I don't see any changes.

Reply:
Sorry, I missed this somehow, fixed now.